src/activities/invoke-prompt-activities.ts

```diff
@@ -10,18 +10,28 @@ type GetRelatedDocumentsInput = {
   query: string
   latestDocumentProcessingId: string
   s3Bucket: string
+  tossError?: boolean
 }
+
 type GetRelatedDocumentsOutput = {
   conversationFilename: string
 }
+
 export async function generatePrompt(input: GetRelatedDocumentsInput): Promise<GetRelatedDocumentsOutput> {
-  const { query, latestDocumentProcessingId, s3Bucket } = input
+  const { query, latestDocumentProcessingId, s3Bucket, tossError } = input
 
   const pgVectorStore = await getPGVectorStore()
   const results = await pgVectorStore.similaritySearch(query, 5, {
     workflowId: latestDocumentProcessingId
   });
 
+  if (tossError) {
+    const randomErr = Math.random()
+    if (randomErr < 0.6) {
```

Comment:
Instead of `tossError` being a boolean, can it be this percentage? Default it to 0 and allow a value to be passed in to configure how frequently it should fail.